### PR TITLE
Removed uservoice links

### PR DESCRIPTION
**Pull request by M123-dev**
Removed uservoice links 

I hope this is the correct file to edit, the same files need to be updated in openfoodfacts-web too.

Ref: https://openfoodfacts.slack.com/archives/C02KVRT2C/p1616595558021000
CC: @stephanegigandet

### CODE REVIEW

File: lang/en/texts/faq.html [hunk 1]
```diff
@@ -18,7 +16,7 @@ <h1>Frequently Asked Questions</h1>
 
 <h2 id="contact_the_team">How can I contact the Open Food Facts team, ask questions or make suggestions?</h2>
 
-<p>You can ask questions or make suggestions on the <a href="https://openfoodfacts.uservoice.com/">Idea Forum</a> or contact us by e-mail: contact@openfoodfacts.org</p>
+<p>You can ask questions or make suggestions on the <a href= "https://slack-openfoodfacts.herokuapp.com/">Slack chatroom</a> or contact us by e-mail: contact@openfoodfacts.org</p>
```
Comment (stephanegigandet) [inline lang/en/texts/faq.html:19]:
```suggestion
<p>You can ask questions or make suggestions on the <a href= "https://slack.openfoodfacts.org/">Slack chatroom</a> or contact us by e-mail: contact@openfoodfacts.org</p>

```

**Review by stephanegigandet — APPROVED** [review]
Looks good to me! @teolemon ?

**Comment by teolemon** [comment]
I guess so. Collecting feedback is important though.

**Comment by M123-dev** [comment]
Yeah you're right, 

in the weekly meeting where you both had another meeting we talked about something similar and came to the conclusion that something like a forum would probably be good, this would also be good for feedback.
@CharlesNepote any progress there

**Review by hangy — APPROVED** [review]

**Comment by M123-dev** [comment]
From my side this pr is done, but I don't have the rights to merge on this repo

**Pull request merged**